schema/base/landUse.yaml

```diff
@@ -0,0 +1,159 @@
+        - campground
+        - cemetary
+        - conservation
+        - construction
+        - developed
+        - education
+        - entertainment
+        - golf
+        - horticulture
+        - landfill
+        - medical
+        - military
+        - park
+        - public
+        - protected
+        - recreation
+        - religious
+        - residential
+        - resourceExtraction
+        - structure
+        - transportation
+        - winterSports
+      class:
+        description: Further classification of the landuse
+        type: string
+        enum:
+          - aboriginalLand
+          - aerodrome
+          - airfield
+          - allotments
+          - animalKeeping
+          - aquaculture
+          - barracks
+          - base
+          - brownfield
+          - bunker
+          - campSite
+          - cemetery
+          - churchyard
+          - civicAdmin
+          - clinic
+          - college
+          - commercial
+          - common
+          - conservation
+          - construction
+          - dam
+          - dangerArea
+          - depot
+          - doctors
+          - dogPark
+          - drivingRange
+          - education
+          - environmental
+          - fairway
+          - farmland
+          - farmyard
+          - flowerbed
+          - forest
+          - garages
+          - garden
+          - golfCourse
+          - grass
+          - green
+          - greenfield
+          - greenhouseHorticulture
+          - helipad
+          - heliport
+          - highway
+          - hospital
+          - industrial
+          - institutional
+          - landfill
+          - lateralWaterHazard
+          - logging
+          - marina
+          - meadow
+          - military
+          - militaryOther
+          - nationalPark
+          - naturalMonument
+          - natureReserve
+          - navalBase
+          - nuclearExplosionSite
+          - obstacleCourse
+          - orchard
+          - park
+          - peatCutting
+          - pier
+          - pitch
+          - plantNursery
+          - playground
+          - protectedLandscapeSeascape
+          - public
+          - quarry
+          - range
+          - recreationGround
+          - religious
+          - residential
+          - retail
+          - rough
+          - saltPond
+          - school
+          - schoolyard
+          - speciesManagementArea
+          - stadium
+          - statePark
+          - staticCaravan
+          - strictNatureReserve
+          - tee
+          - themePark
+          - track
+          - trafficIsland
+          - trainingArea
+          - trench
+          - university
+          - villageGreen
+          - vineyard
+          - waterHazard
+          - waterPark
+          - wildernessArea
+          - winterSports
+          - zoo
+      names: { "$ref": "../defs.yaml#/$defs/propertyDefinitions/names" }
+      surface:
+        description: Surface material
```

Comment:
Wondering if this should mention if it is coming directly from some OSM tag as-is, otherwise it should probably have enum.

Reply:
Good point, we do _some_ normalization to this tag.